src/Symfony/Component/DependencyInjection/Tests/Config/ContainerParametersResourceCheckerTest.php

```diff
@@ -62,11 +62,11 @@ public function isFreshProvider()
         }, false];
 
         yield 'fresh on every identical parameters' => [function (MockObject $container) {
-            $container->expects($this->exactly(2))->method('hasParameter')->willReturn(true);
-            $container->expects($this->exactly(2))->method('getParameter')
+            $container->expects(self::exactly(2))->method('hasParameter')->willReturn(true);
```

Comment:
You've changed the calls inside this method to static, but don't we need to change the declaration of `isFreshProvider()` to `static` as well?

Reply:
Forgot to push 😄